src/styles/header.scss

```
@@ -0,0 +1,67 @@
+@import 'variables';
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px 0 #0000000d;
+}
+
+.logo {
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+.logo__image {
+  height: 100%;
+  width: 100%;
+}
+
+.nav__list {
+  display: flex;
+  padding: 0;
+  margin: 0;
+  list-style: none;
+}
+
+.nav__item {
+  margin-left: 20px;
+}
+
+.nav__item:first-child {
+  margin-left: 0;
+}
+
+.nav__link {
+  position: relative;
+  display: flex;
+  align-items: center;
+  height: 60px;
+  font-size: 12px;
+  font-weight: 500;
+  white-space: nowrap;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: $primary-color;
+}
+
+.nav__link:hover {
+  color: $accent-color;
+}
```

```suggestion
.nav__link {
  position: relative;
  display: flex;
  align-items: center;
  height: 60px;
  font-size: 12px;
  font-weight: 500;
  white-space: nowrap;
  text-decoration: none;
  text-transform: uppercase;
  color: $primary-color;
&:hover {
  color: $accent-color;
}
}
```
pls make use of SASS BEM nesting
**PLS DO THE SAME FOR ALL SUCH CASES IN THE PROJECT**